DXMainClient/PreStartup.cs

```diff
@@ -207,17 +255,17 @@ private static void CheckPermissions()
         /// <param name="accessRights">The file system rights.</param>
         private static bool UserHasDirectoryAccessRights(string path, FileSystemRights accessRights)
         {
-#if WINDOWSGL
             // Mono doesn't implement everything necessary for the below to work,
             // so we'll just return to make the client able to run on non-Windows
             // platforms
             // On Windows you rarely have a reason for using the OpenGL build anyway
```

Comment:
I would say that this comment needs reviewing as this statement is not really true.